zingolib/src/commands.rs

```diff
@@ -97,13 +97,13 @@ impl Command for WalletKindCommand {
 
     fn exec(&self, _args: &[&str], lightclient: &LightClient) -> String {
         RT.block_on(async move {
-            if lightclient.do_seed_phrase().await.is_ok() {
+            if lightclient.get_wallet_kind().await.is_ok() {
```

Comment:
This doesn't seem like a well-defined fn for publishing whether a wallet is view-key only.

Reply:
I am +1 on the logical flow of this exec implementation.